### PR TITLE
add tide query to require group-lead-approved label in realease-4.5

### PR DESCRIPTION
**Pull request by droslean**
Requested in [DPTP-1121](https://issues.redhat.com/browse/DPTP-1121)

/hold

This needs to be merged after the branch cut. Related PR https://github.com/openshift/release/pull/8535

/cc @jupierce @stevekuznetsov @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -641,7 +641,6 @@ tide:
     - release-4.2
     - release-4.4
     - release-4.3
-    - release-4.5
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml:644]: Not this one. `ocs-operator` is AFAIK not involved in openshift processes yet, so we do not need to touch their queries

File: core-services/prow/02_config/_config.yaml [hunk 2]
```diff
@@ -2231,6 +2228,208 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - opendatahub-io/odh-manifests
+  - includedBranches:
+    - release-4.5
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml:2232]: This should include also the `openshift-4.5` branch (which also needs to be removed from the queries above that include it)

File: core-services/prow/02_config/_config.yaml [hunk 3]
```diff
@@ -2231,6 +2228,208 @@ tide:
...
+    labels:
+    - group-lead-approved
+    - lgtm
+    - approved
+    - bugzilla/valid-bug
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml:2237]: Just to be sure, `cherry-pick-approved` is not required intentionally?

Comment (droslean) [reply core-services/prow/02_config/_config.yaml:2237]: Yes. We will enable cherry-pick-approved when the GA comes.

**Pull request merged**